zebra-rpc/src/methods/types/get_blockchain_info.rs

```diff
@@ -0,0 +1,72 @@
+//! Types used in `getblockchaininfo` RPC method.
+
+use zebra_chain::{
+    amount::{Amount, NonNegative},
+    value_balance::ValueBalance,
+};
+
+use super::*;
+
+/// A value pool's balance in Zec and Zatoshis
+#[derive(Clone, Debug, Eq, PartialEq, serde::Serialize, serde::Deserialize)]
+#[serde(rename_all = "camelCase")]
+pub struct ValuePoolBalance {
+    /// Name of the pool
+    id: String,
+    /// Total amount in the pool, in ZEC
```

Comment:
Do we want to include `monitored` here? It would always be true in Zebra.

Reply:
Yea, let's don't add it by now.